### ISSUE TITLE
Gui is not runnning

### ISSUE THREAD

**Issue by gasilva** [ajaiantilal/i7z#1]
I have a dual socket Xeon sandy-bridge. The i7z is work but not the GUI. Both compiled with no errors. I already installed msrtools and msr-tools in ubuntu 12.10. Could you give me some light on this issue?

``` bash
i7z DEBUG: i7z version: svn-r77-(20-Nov-2011)
i7z DEBUG: Found Intel Processor
i7z DEBUG:    Stepping 7
i7z DEBUG:    Model d
i7z DEBUG:    Family 6
i7z DEBUG:    Processor Type 0
i7z DEBUG:    Extended Model 2
i7z DEBUG: msr = Model Specific Register
i7z DEBUG: Detected an i7 - 32nm (haven't seen this version around, do write to me with the model number)
i7z DEBUG: msr device files exist /dev/cpu/*/msr
i7z DEBUG: You have write permissions to msr device files
```

**Comment by ajaiantilal** [comment]
hello 

am i correct to assume that the console version works (the ncurses gui), and not the Qt-based GUI version. 

can you paste the entire output that you get?

**Comment by gasilva** [comment]
Yes. The text-based version is working OK. I just dropped this note to inform you about that.

I compiled and ran the code in an i7 and both text and GUI versions worked in Ubuntu 12.04. The GUI did not work in a Xeon with Ubuntu 12.10.

The code lines above are all the output I got after the crash.

**Comment by ajaiantilal** [comment]
thanks a lot for the bug report.

yeh, the qt code has some _major_ issues going on. some combination like using optimization flags seem to crash it. i am embarrassed to say that even it doesn't display the bars correctly on my development machine :(

i am thinking of doing a major rehaul of the qttool once i get some more time to work on it (and test on multiple OS's) in maybe a couple of weeks.
